types/react-table-config.d.ts

```diff
@@ -49,11 +49,13 @@ import {
 } from 'react-table';
 
 declare module 'react-table' {
+  type Align = { align?: 'left' | 'center' | 'right' };
   export interface ColumnInstance<
     D extends Record<string, unknown> = Record<string, unknown>
   > extends UseFiltersColumnProps<D>,
       UseGroupByColumnProps<D>,
       UseResizeColumnsColumnProps<D>,
+      Align,
```

Comment:
Seems we can add anything into the columns we like which is nice. But the fields 1) have to be optional 2) need to be defined in a type explictly.

i.e.
```
...
UseResizeColumnsColumnProps<D>,
{ align: 'left' | 'center' | 'right' },
...
```

didn't work hehe